Documentation/Signoffs/Mario Kart Rev 2 - Signoff - Mario Kart Bike Wireless Sensors Subsystem.md

```diff
@@ -0,0 +1,171 @@
+# Mario Kart Rev 2 - Signoff - Mario Kart Bike Wireless Sensors Subsystem 
+## Team 4 - Blake Pickett ##
+
+_Function of the Subsystem:_
+
+The function of the newly designed feature is to enable Bluetooth LE (BLE) connectivity between sensors and central Raspberry Pi (RPi). These sensors will gather, convert, and send data via BLE to the master RPi. The wires that rectify and step down the power for the sensors on the current version of the Mario Kart Bike will be replaced with four D batteries and a holder. The four D batteries will provide 6 V and 2.5 A to the microcontrollers, which will power the sensors for six hours of constant use. 
+
+_Constraints:_ 
+
+**Safety:**
+
+The long wires are a trip hazard. The long wires must be replaced with batteries. 
+
+**Run Time:**
+
+Multiple batteries must supply the DC voltage and current needed to power the equipment for 4 hours of constant use a day for 2 weeks, which is 56 hours of constant use. 
```

Comment:
This doesn't consider the battery usage in the remaining 20*14 hours of idle time.